examples/get-started/contour-map.py

```diff
@@ -0,0 +1,130 @@
+# at :gmt-docs:`cookbook/cpts.html`.
+
+fig = pygmt.Figure()
+fig.grdimage(grid=grid, frame="a", projection="M10c", cmap="haxby")
+fig.show()
+
+###############################################################################
+# Adding a colorbar
+# -----------------
+#
+# To show how the plotted colors relate to the Earth relief, a colorbar can be
+# added using the :meth:`pygmt.Figure.colorbar` method.
+#
+# To control the labels on the colorbar, a list is passed to the ``frame``
+# parameter. The value beginning with "a" sets the interval for annotation on
+# the colorbar, in this case every 1,000 meters. To set the label for an axis
+# on the colorbar, the value begins with either "x+l" (x-axis) or "y+l"
+# (y-axis), followed by the intended label.
+#
+# By default, the CPT for the colorbar is the same as the one set
+# in :meth:`pygmt.Figure.grdimage`.
+
+fig = pygmt.Figure()
+fig.grdimage(grid=grid, frame="a", projection="M10c", cmap="haxby")
+fig.colorbar(frame=["a1000", "x+lElevation", "y+lm"])
+fig.show()
+
+###############################################################################
+# Adding contour lines
+# --------------------
+#
+# To add contour lines to the color coded figured, the
+# :meth:`pygmt.Figure.grdcontour` method is used. The ``frame`` and
+# ``projection`` are already set using :meth:`pygmt.Figure.grdimage` and are
+# not needed again. However, the same input for ``grid`` (in this case, the
+# variable named "grid") must be input again. The ``interval`` parameter sets
+# the spacing between lines (in this case, 500 meters), and the ``annotation``
+# parameter draws darker lines that are annotated with the elevation or
+# bathymetry.
+
+fig = pygmt.Figure()
+fig.grdimage(grid=grid, frame="a", projection="M10c", cmap="haxby")
+fig.grdcontour(grid=grid, interval=500, annotation=1000)
+fig.colorbar(frame=["a1000", "x+lElevation (m)"])
+fig.show()
+
+###############################################################################
+# Color in land
+# -------------
+#
+# To make it clear where the islands are located, the
+# :meth:`pygmt.Figure.coast` method can be used to color in the land. The
+# ``land`` is colored in as "lightgray", and the ``shorelines`` parameters
+# draws a border around the islands.
+
+fig = pygmt.Figure()
+fig.grdimage(grid=grid, frame="a", projection="M10c", cmap="haxby")
+fig.grdcontour(grid=grid, interval=500, annotation=1000)
+fig.coast(shorelines="2p", land="lightgray")
+fig.colorbar(frame=["a1000", "x+lElevation (m)"])
+fig.show()
+
+###############################################################################
+# Additional exercises
+# --------------------
+#
+# This is the end of the second tutorial. Here are some additional exercises
+# for the concepts that were discussed:
+#
+# 1. Change the resolution of the grid file to either "01m" (lower resolution)
+#    or "15s" (higher resolution). Note that higher resolution grids will have
+#    larger file sizes.
```

Comment:
Maybe link to https://www.generic-mapping-tools.org/remote-datasets/earth-relief.html#usage so users know what are the possible spatial resolutions, and mention explicitly that `01m` stands for 1 arc minute (some people might think it's 1 meter).